### PR TITLE
AL-461: display total modal

### PR DESCRIPTION
**Pull request by benoitgrasset-alma**
### Ticket
https://app.clickup.com/t/20427503/AL-461

### Screenshot

modal-total

### CODE REVIEW

**Comment by gautierlabarre** [comment]
Capture d’écran 2022-05-05 à 16 25 39

Little issue with responsiveness. Also it would be nice to add a little margin-bottom after your summary to give more space with the text below

File: src/Widgets/EligibilityModal/components/Schedule/index.tsx
Comment (gautierlabarre) [inline src/Widgets/EligibilityModal/components/Schedule/index.tsx]: Can you add a new class in the className.const.ts (STATIC_CUSTOMISATION_CLASSES) ? It's for the merchant to customize this element if need be ?

Comment (benoitgrasset-alma) [reply src/Widgets/EligibilityModal/components/Schedule/index.tsx]: done, class added, and I also fixed CSS issue

Comment (gautierlabarre) [reply src/Widgets/EligibilityModal/components/Schedule/index.tsx]: You added a margin: 24px 0; which add a margin-top and margin-bottom to 24px. I'm ok with this, but in this case reduce the padding from .schedule (  margin-bottom: 40px; into 16px).

Anyway i'm a big confused between figma and the ticket you had on "un crédit vous engage..." in figma it's above the total and in your ticket you were asked to do it in the total. I don't know which is the source of truth.

So it's good for me :)

**Review by gautierlabarre — APPROVED** [review]

File: src/Widgets/EligibilityModal/components/Schedule/index.tsx
Comment (benoitgrasset-alma) [reply src/Widgets/EligibilityModal/components/Schedule/index.tsx]: You're rigth :) I reduced it

**Review by murdink — CHANGES_REQUESTED** [review]
Alma Widgets basic example 2022-05-06 16-38-57
IMO this text should be aligned to the right `text-align: right` so it looks good on smaller screens
image

Apart from that looking good to me

**Comment by murdink** [comment]
Also edges are supposed to be around, I think you forgot the border-radius :)
image
Edit: The gray color is so light I wonder if it even makes sense, I fear it won't even show up on brighter / hight contrast screens - maybe it's worth challenging - maybe we want to use an existing darker shade

File: src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css
Comment (murdink) [inline src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css]: as mentioned we want to use CSS variables

Comment (benoitgrasset-alma) [reply src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css]: good point, it is fixed

**Comment by benoitgrasset-alma** [comment]
Hi @murdink waiting for your approval :)

**Review by murdink — APPROVED** [review]

File: src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css
Comment (benoitgrasset-alma) [inline src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css:25]: @murdink here is the border-radius :)

**Pull request merged**
